app/services/idv/steps/in_person/address_step.rb

```diff
@@ -21,7 +21,8 @@ def call
         def extra_view_variables
           {
             pii: flow_session[:pii_from_user],
-            updating_address: flow_session[:pii_from_user].has_key?(:address1),
+            updating_address: flow_session[:pii_from_user].has_key?(:address1) &&
+              !flow_session[:pii_from_user][:address1].nil?,
```

Comment:
i'm unclear about the additional check - are we ensuring that after updating the address to additional include the secondary one that the user is not clearing the `address1` line from their original state id address submission?